API/Extensions/ScopeRequirement.cs

```diff
@@ -50,10 +52,19 @@ public ScopeRequirementHandler(IHttpContextAccessor httpContextAccessor, IUserSe
         /// </summary>
         /// <param name="context">The authorization context.</param>
         /// <param name="requirement">The requirement to evaluate.</param>
-        /// <returns></returns>
+        /// <returns>CompletedTask.</returns>
         protected override Task HandleRequirementAsync(AuthorizationHandlerContext context, ScopeRequirement requirement)
         {
-            string identityId = httpContextAccessor.HttpContext.User.GetIdentityId(httpContextAccessor.HttpContext);
+            string identityId;
+            try
+            {
+                identityId = httpContextAccessor.HttpContext.User.GetIdentityId(httpContextAccessor.HttpContext);
+            } catch(UnauthorizedAccessException e)
+            {
+                Log.Logger.Error(e, "User is not authenticated.");
```

Comment:
Are we sure we want to log this? This would trigger Sentry. 
I'm worried about possible spam and convolution the logs.

Reply:
I think for debug wise it would be good to log it either way but is there a way to exclude it from sentry? Either a specific exclusion or unautorized exclusion?

Reply:
We can lower the severity of the error? That way it does not get logged.

Reply:
After discussing this with Rene, we decided it should be Information instead of Error. I made a quick commit to resolve this.